cards/abdominal-pain-diagnostics/card.md

```diff
@@ -10,6 +10,7 @@ categories:
     - Surgery
     - EBM
 drugs: null
+pubmed: (Diagnosis/Broad[filter]) AND (Imaging and laboratory testing in acute abdominal pain.) 
```

Comment:
Those are the PubMed search terms we discussed adding for future updates. Trying out a separate line for `pubmed` and `google scholar`